.pylintrc

```diff
@@ -211,7 +211,6 @@ enable=abstract-class-instantiated,
     return-arg-in-generator,
     return-in-init,
     return-outside-function,
-    round-builtin,
```

Comment:
https://stackoverflow.com/questions/59533912/what-is-the-alternative-to-built-in-round

The warning screams at any usage of `round` because the implementation changed from py2 to py3 and it might cause transition problems.

Reply:
If you'd like to open a separate PR with just this change, I'd merge it right away.